stub/src/test/java/io/grpc/stub/ClientCallsTest.java

```diff
@@ -74,17 +66,29 @@
 import java.util.concurrent.ExecutionException;
 import java.util.concurrent.Semaphore;
 import java.util.concurrent.TimeUnit;
+import org.junit.After;
```

Comment:
This is going to break checkstyle. Getting onto the new import scheme is https://github.com/grpc/grpc-java/issues/2038